instance/tests/models/test_openedx_database_mixins.py

```diff
@@ -74,7 +74,8 @@ def _assert_privileges(self, database):
             # Pass password using MYSQL_PWD environment variable rather than the --password
             # parameter so that mysql command doesn't print a security warning.
             env = {'MYSQL_PWD': password}
-            mysql_cmd = "mysql -u {user} -e 'SHOW TABLES' {db_name}".format(user=user, db_name=database_name)
+            mysql_cmd = "mysql -h 127.0.0.1 -u {user} -e 'SHOW TABLES' {db_name}".format(user=user,
```

Comment:
Same comment as above – will this work on Vagrant?

Reply:
Tested on vagrant and works.